mmv1/third_party/terraform/utils/node_config.go.erb

```diff
@@ -937,3 +959,31 @@ func flattenLinuxNodeConfig(c *container.LinuxNodeConfig) []map[string]interface
 	return result
 }
 <% end -%>
+
+func expandContainerAdvancedMachineFeatures(v interface{}) *container.AdvancedMachineFeatures {
+	if v == nil {
+		return nil
+	}
+	ls := v.([]interface{})
+	if len(ls) == 0 {
+		return nil
+	}
+	amf := &container.AdvancedMachineFeatures{}
+
+	cfg := ls[0].(map[string]interface{})
+
+	if v, ok := cfg["threads_per_core"]; ok {
+		amf.ThreadsPerCore = int64(v.(int))
+	}
+
+	return amf
+}
+
+func flattenContainerAdvancedMachineFeatures(AdvancedMachineFeatures *container.AdvancedMachineFeatures) []map[string]interface{} {
```

Comment:
nit: mind renaming `AdvancedMachineFeatures` to a lowercase name? I'd recommend `c` for consistency with other flatteners in the file (not sure why they're all called that tbh- probably for "config" or "container") or `adm`. Capital letters indicate exported values, for the most part, which this isn't.